### ISSUE TITLE
Action buttons may wrap on Scores tab

### ISSUE THREAD

**Issue by poelstra** [FirstLegoLeague/fllscoring#163]
Reported by Rob: especially on lower res screens, clicking Edit on a score may cause the action buttons to wrap, in turn causing the edited score to be scrolled out of view.

See #162.

**Comment by rikkertkoppes** [comment]
Proposal (restyling of scores page):

- edit in a dialog, gets rid of the delete button also
- something for publish / unpublish to get rid of that button
- refresh in one button, in header
- update to material design lists

I'll start working on this in a branch

**Comment by poelstra** [comment]
Good idea, although a quick fix for now may be to simply mark the Actions column as nowrap.

**Comment by rikkertkoppes** [comment]
That would push them off screen, which is worse

**Comment by poelstra** [comment]
Hmyeah, true, for very small screens. Looking at what Rob was using I think it might have done the trick.
Anyway, it's an ugly hack too, agreed.
